test/json-schema.js

```diff
@@ -310,25 +310,6 @@ describe('Swagger Schema to JSON Schema', () => {
       });
     });
 
-    it('dereferences root reference and converts to JSON Schema', () => {
```

Comment:
This test is no longer applicable, since the referenced document is no longer being converted as the source is already converted to JSON Schema. Removing the "converts to JSON Schema" part of the test is already the same test as the test case above so becomes redundant.